### PR TITLE
Update primo-alert.v2.json

### PR DESCRIPTION
**Pull request by laura-ake**
Banner update for the application of "November Update" Dec. 17

### CODE REVIEW

**Comment by laura-ake** [comment]
Not sure why this didn't work

File: js/primo-alert.v2.json [hunk 1]
```diff
@@ -4,11 +4,11 @@
         "target element": "#exlidMainMenuTile",
         "logged in": false,
         "patron status": ["01", "02", "03", "10", "20", "23", "04", "25", "24"],
-        "heading": "Direct Request for Georgia Tech items; plus, DiscoverE has a new style",
-        "message": "<p>Direct requesting for Georgia Tech items in discoverE is now available. Please contact the <a href=\"http://emory.libanswers.com\">Library Service Desk<\/a> if you need assistance.</p> <p>You may notice some improvements to the styling of discoverE; basic functionality is the same. Please feel free to <a href=\"http:\/\/web.library.emory.edu\/research-learning\/research-tools\/discoverE\/discovere-feedback.html\">leave a comment or report a problem</a>.</p>"
+        "heading": "DiscoverE update",
+        "message": "<p>DiscoverE maintenance and brief outage Nov. 17 2017 between 1 a.m. and 6 a..m.<b><a href=\"https://scholarblogs.emory.edu/librarycoreservices/2017/12/14/discovere-update-12-17-2017/\">More information here.</a></b></p>
```
Comment (eipeele) [inline js/primo-alert.v2.json]: This should be "message": "<p><a href=\"https://scholarblogs.emory.edu/librarycoreservices/2017/12/14/discovere-update-12-17-2017/\">DiscoverE maintenance and brief outage Nov 17 2017 between 1 am and 6 am.</a></p>" so screenreaders can more easily read.

Comment (laura-ake) [reply js/primo-alert.v2.json]: Thanks that's fine.

**Comment by bernardo-gomez** [comment]
laura, approved your change.

**Pull request closed without merging**